### ISSUE TITLE
enterText(_:intoViewWithAccessibilityLabel:) sometimes fails to enter last characters

### ISSUE THREAD

**Issue by sdduursma** [kif-framework/KIF#725]
`enterText(_:intoViewWithAccessibilityLabel:)` sometimes fails to enter the last characters of the given text, giving the following error: `The step timed out after 1.00 seconds: Failed to get text "Cash" in field; instead, it was "Cas"`.

Xcode 7.0, iOS 9

**Comment by phatmann** [comment]
I am sorry to see this bug come back. I thought it had been squashed by 552f4412. Do you see this every time or just sometimes?

**Comment by sdduursma** [comment]
Only sometimes.

**Comment by phatmann** [comment]
This is a bad issue for sure, but the entire KIF community has failed to solve it after numerous attempts. It sporadic nature makes it difficult to debug. Do you have a scenario where it happens often enough to diagnose?

**Comment by sdduursma** [comment]
Unfortunately not. It seems to happen more often on our CI than locally. The CI uses a slow machine so that might contribute to the failure.

**Comment by mikelupo** [comment]
A stab in the dark. What if before you execute that line, you set the timeout to something higher than 10?
[KIFUITestActor setDefaultTimeout: 30];


Don't forget to set it back to 10 (or whatever your normal timeout is)



-----Original Message-----

**Comment by sdduursma** [comment]
Since I posted this issue I haven't seen this happen again. Very weird. It was happening quite frequently at the time but never since.

**Comment by satchnut** [comment]
Looks like a case of Heggs' law ("Luther Heggs" from The Ghost and Mr Chicken <http://www.imdb.com/title/tt0059221/>)

**Comment by phatmann** [comment]
@sdduursma thanks for the update. I am going to cautiously and optimistically close this issue. If you start seeing it again please re-open it.
